fe/__tests__/components/FailureMask/FailureMask_spec.re

```diff
@@ -0,0 +1,13 @@
+open Jest;
+
+let () =
```

Comment:
dziwnie to wygląda. W leonie piszemy testy zaczynając od describe, tutaj można to jeszcze opakować ?